devsite/devsite/celery.py

```diff
@@ -8,13 +8,13 @@
 from django.conf import settings
 
 
-CELERY_CHECK_MSG_PREFIX = 'figures-devsite-celery-check'
+CELERY_CHECK_MSG_PREFIX = "figures-devsite-celery-check"
```

Comment:
Is there a problem with the single quotes?

Reply:
No, I had black enabled at some point. I have disabled it since but looks like I missed some of the changes that were made.

Reply:
Thanks. Appreciate that!